numba/_lapack.c

```diff
@@ -1,9 +1,4 @@
 /*
- * This file contains wrappers of BLAS and LAPACK functions
```

Comment:
Sorry, for some reason I (admittedly mindlessly) thought that comment applied to the following <complex.h> inclusion.